### ISSUE TITLE
Indicate default value of propose.by.prior for initializeParameterObject in parameterObject.R (and c++ code) and how the values are generated if set to false.

### ISSUE THREAD

**Issue by mikegilchrist** [acope3/RibModelFramework#340]

**Comment by mikegilchrist** [comment]
Indicate default value of propose.by.prior for initializeParameterObject in parameterObject.R (and c++ code) and how the values are generated if set to FALSE.

**Comment by mikegilchrist** [comment]
Also need to indicate when this might be useful to set to TRUE and what happens if we are running an adaptive MCMC.

**Comment by acope3** [comment]
Cedric is skeptical of the utility of this. Letting each \DeltaM value have its own prior could be handy in cases where empirical data on mutation biases are available. However, Cedric thinks proposing based on these values defeats the point of the MCMC: exploring the parameter space for \DeltaM will be more restricted. I can see this being especially problematic if the prior is informed by poor empirical data. Prior to discussing things with Cedric, I was already considering removing this functionality.

**Comment by mikegilchrist** [comment]
I disagree, especially for removing it if it already exists.  Designating codon specific priors would allow us to test hypotheses about mutation.

**Comment by clandere** [comment]
> I disagree, especially for removing it if it already exists. Designating codon specific priors would allow us to test hypotheses about mutation.

As I said, and alex stated, it could be handy to have a prior for individual deltaM. However, that is not what this functionality is. Here the idea is to propose new values based on the prior. Turning the prior into the proposal distribution. This defeats the purpose of an MCMC in my opinion.
